controllers/provisioners/eks/create.go

```diff
@@ -62,6 +62,14 @@ func (ctx *EksInstanceGroupContext) Create() error {
 		configName = fmt.Sprintf("%v-%v", ctx.ResourcePrefix, common.GetTimeString())
 	}
 
+	if configuration.Image == "latest" {
```

Comment:
can we make latest a constant?

Reply:
Also good if you can use `strings.EqualFold` to avoid case sensitivity issues